core/components/atoms/avatarGroup/AvatarGroup.tsx

```diff
@@ -117,13 +131,23 @@ export const AvatarGroup = (props: AvatarGroupProps) => {
   );
 
   return (
-    <div data-test="DesignSystem-AvatarGroup" {...baseProps} className={`${AvatarGroupClass} d-inline-flex`}>
-      <Avatars avatarList={list.slice(0, max)} avatarStyle={style} tooltipPosition={tooltipPosition} />
+    <div
+      data-test="DesignSystem-AvatarGroup"
+      style={{ color: 'inherit' }}
```

Comment:
Please try to remove this `style` attribute and achieve the same using classname

Reply:
done